composer.json

```diff
@@ -67,6 +67,12 @@
 		"check-cs": [
 			"@php ./vendor/squizlabs/php_codesniffer/scripts/phpcs"
 		],
+		"premium-check-cs": [
+			"composer require yoast/yoastcs:~1.2.1 --update-with-dependencies --no-suggest --no-interaction",
```

Comment:
Will running `composer check-cs` break when being run on the Premium repo, because of the testVersion being set to 5.2-?

Reply:
The `composer check-cs` command will "break" no matter what as it would be using the outdated versions of PHPCS/WPCS/YoastCS and because of that, will:
* Not recognize the Premium specific ruleset and therefore use the ruleset intended for Free only instead.
* Throw false positives for bugs which have been fixed in newer PHPCS/WPCS versions.
* Throw false positives on the lines being ignored by the new-style annotations as they aren't recognized yet.
* Throw false negatives on all the rules which have been added since the outdated versions, but are not being tested.